sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretModelFactory.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Security.KeyVault.Secrets
```

Comment:
Part of me still thinks maybe these should go in a different namespace. Possibly Azure.Security.KeyVault.Secrets.Mocking or Testability. Thoughts?

Reply:
@KrzysztofCwalina or @pakrym, thoughts to changing that? It does muddy the waters to put it in the same namespace.

Reply:
I don't have a strong opinion but namespace with a single type would look strange and VS 19 shows types from all namespaces anyway.

Reply:
AppConfig and Storage are putting them in the same namespace, so I'll leave as-is for consistency.